platform/jobs/edxPlatformPythonUnitTestsPr.groovy

```diff
@@ -274,17 +286,20 @@ jobConfigs.each { jobConfig ->
            archiveJunit(JENKINS_PUBLIC_JUNIT_REPORTS) {
                retainLongStdout()
            }
-           publishHtml {
-               report("${jobConfig.repoName}/reports") {
-                   reportFiles('diff_coverage_combined.html')
-                   reportName('Diff Coverage Report')
-                   keepAll()
+           // Only archive Coverage data when explicitly defined in the jobConfig to avoid build errors
+           if (jobConfig.runCoverage) {
+               publishHtml {
+                   report("${jobConfig.repoName}/reports") {
+                       reportFiles('diff_coverage_combined.html')
+                       reportName('Diff Coverage Report')
+                       keepAll()
+                   }
                }
-           }
-           configure { node ->
-               node / publishers << 'jenkins.plugins.shiningpanda.publishers.CoveragePublisher' {
+               configure { node ->
+                   node / publishers << 'jenkins.plugins.shiningpanda.publishers.CoveragePublisher' {
+                   }
                }
-           }
+            }
```

Comment:
nit: does this have an extra space in there now?

Reply:
indeed. Cleaned it up